core/src/mindustry/world/blocks/defense/turrets/Turret.java

```diff
@@ -277,6 +278,14 @@ public void updateTile(){
         @Override
         public void drawSelect(){
             Drawf.dashCircle(x, y, range, team.color);
+            if(target != null && Core.settings.getBool("blockstatus")){
+                float time = Time.time();
+                Lines.stroke(3f, Pal.gray);
+                Lines.square(target.getX(), target.getY(), 8 + 1f + Mathf.sinDeg(time*2), 45+time);
+                Lines.stroke(1f, Team.crux.color);
+                Lines.square(target.getX(), target.getY(), 8 + 1f + Mathf.sinDeg(time*2), 45+time);
+                Draw.reset();
+            }
```

Comment:
The `float time = Time.time();` isn't nessesarry.
```suggestion
            if(target != null && Core.settings.getBool("blockstatus")){
                Lines.stroke(3f, Pal.gray);
                Lines.square(target.getX(), target.getY(), 8 + 1f + Mathf.sinDeg(Time.time() * 2), 45 + Time.time());
                Lines.stroke(1f, Team.crux.color);
                Lines.square(target.getX(), target.getY(), 8 + 1f + Mathf.sinDeg(Time.time() * 2), 45 + Time.time());
                Draw.reset();
            }
```

Reply:
I actually did it to avoid millisecond lag